src/main/scala/com/cognite/sdk/scala/v1/threeD.scala

```diff
@@ -60,8 +60,8 @@ final case class ThreeDRevisionUpdate(
 final case class ThreeDAssetMapping(
     nodeId: Long,
     assetId: Long,
-    treeIndex: Long = 0,
-    subtreeSize: Long = 0
+    treeIndex: Option[Long],
```

Comment:
So it's still present sometimes, right, not removed entirely?

Reply:
Yes, the usual case is that all 4 fields will be there. I am not sure the practical use-case of an asset mapping connected to a node that does not exist, but only in that case the treeIndex and subtreeSize would be None.

Reply:
I should have read the PR description. ;)